src/Raven.Studio/typescript/common/shell/menu/items/rootItems.ts

```diff
@@ -35,30 +35,24 @@ function aboutItem() {
 }
 
 interface WhatsNewItemOptions {
-    isNewVersionAvailable?: boolean;
     isWhatsNewVisible?: boolean;
 }
 
-function whatsNewItem({ isNewVersionAvailable = false, isWhatsNewVisible = false }: WhatsNewItemOptions = {}) {
+function whatsNewItem({ isWhatsNewVisible = false }: WhatsNewItemOptions = {}) {
```

Comment:
Let's also remove `isNewVersionAvailable` from `shell.ts`, `generateMenuItems.ts` and `StudioSearchWithDatabaseSwitcher.stories.tsx`

Reply:
done